example/android/gradle.properties

```
@@ -22,3 +22,4 @@ org.gradle.jvmargs=-Xmx1536m
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
+android.enable48.fullMode=true
```

@Subhash703 - is this tested properly??

```suggestion
android.enableR8.fullMode=true
```